.ddev/commands/web/gulp

```diff
@@ -28,5 +28,5 @@ then
   cp ../../eheza-infra/elm/Config.Deploy.elm ${CLIENT_PATH}/src/elm/
 
 fi
-
+export PATH=~/.nvm/versions/node/v9.0.0/bin:$PATH;
```

Comment:
Should not be part of this PR. Remove.

Reply:
> @Diama1 is this ready for review?

Yeah, it took long for the test to pass but now it ready for review